### ISSUE TITLE
Add support for Cordova 6

### ISSUE THREAD

**Issue by daniel-jackson95** [thisisbd/cordova-plugin-vuforia#4]
Currently doesn't work for Cordova 6.

**Comment by mattrayner** [comment]
Cordova has changed the `appdelegate.m` contents - we need to investigate how to add a navigation controller

**Comment by mattrayner** [comment]
We will look into this for a future release - possibly 2.0

**Comment by mattrayner** [comment]
@dsgriffin has come up with the below solution to `corodva@6.0` and `cordova-ios@4.0` support

* Enable Bitcode is already set to no
* Minimum deployment target is iOS 8

---

In `AppDelegate.m`:
First, Cordova CDVPlugin needs to be explicitly imported at the top of the file `#import <Cordova/CDVPlugin.h>`

---

Secondly, didFinishLaunchingWithOptions needs to be replaced with
```
- (BOOL)application:(UIApplication)application didFinishLaunchingWithOptions:(NSDictionary)launchOptions
{
CGRect screenBounds = [[UIScreen mainScreen] bounds];

#if __has_feature(objc_arc)
self.window = [[UIWindow alloc] initWithFrame:screenBounds];
#else
self.window = [[[UIWindow alloc] initWithFrame:screenBounds] autorelease];
#endif
self.window.autoresizesSubviews = YES;
#if __has_feature(objc_arc)
self.viewController = [[MainViewController alloc] init];
#else
self.viewController = [[[MainViewController alloc] init] autorelease];
#endif

// Set your app's start page by setting the <content src='foo.html' /> tag in config.xml.
// If necessary, uncomment the line below to override it.
// self.viewController.startPage = @"index.html";

// NOTE: To customize the view's frame size (which defaults to full screen), override
// [self.viewController viewWillAppear:] in your view controller.

UINavigationController * nc = [[UINavigationController alloc]initWithRootViewController:self.viewController];
[nc setNavigationBarHidden:YES animated:NO];

self.window.rootViewController = nc;
[self.window makeKeyAndVisible];

return YES;
}
```

**Comment by mattrayner** [comment]
Fixed with PR [#17]
